src/main/java/build/buildfarm/backplane/Backplane.java

```diff
@@ -79,7 +79,7 @@ public ActionCacheScanResult(
    * <p>Returns true if the worker was newly added, and false if it was already a member of the set.
    */
   @ThreadSafe
-  boolean addWorker(ShardWorker shardWorker) throws IOException;
+  void addWorker(ShardWorker shardWorker) throws IOException;
```

Comment:
thinking clients should just start checking this for failures to add

Reply:
(sorry for drive-by comment)
Please also update the comment in line 79.